app/src/main/kotlin/com/simplemobiletools/calendar/pro/adapters/DayEventsAdapter.kt

```diff
@@ -89,7 +91,15 @@ class DayEventsAdapter(activity: SimpleActivity, val events: ArrayList<Event>, r
             event_item_title.checkViewStrikeThrough(event.isTaskCompleted())
             event_item_time.text = if (event.getIsAllDay()) allDayString else Formatter.getTimeFromTS(context, event.startTS)
             if (event.startTS != event.endTS && !event.getIsAllDay()) {
-                event_item_time.text = "${event_item_time.text} - ${Formatter.getTimeFromTS(context, event.endTS)}"
+                val startDayCode = Formatter.getDayCodeFromTS(event.startTS)
+                val endDayCode = Formatter.getDayCodeFromTS(event.endTS)
+                val startDate = Formatter.getDayTitle(activity, startDayCode,false)
+                val endDate = Formatter.getDayTitle(activity, endDayCode,false)
```

Comment:
format the code with ctrl + shift + L, there are some missing spaces in both files.